src/scenes/Event/scenes/oracle.js

```diff
@@ -468,7 +469,7 @@ export default class OraclePage extends Component {
     const { locale, messages: localeMessages } = this.props.intl;
     const intl = getIntlProvider(locale, localeMessages);
 
-    this.props.setAppLocation(AppLocation.bet);
+    if (!this.props.appLocation) this.props.setAppLocation(AppLocation.bet);
```

Comment:
why add this check if its defined or not?

Reply:
Basically, we only need it to set appLocation for a specific oracle when we go directly to an oracle. aka refresh the page on an oracle or someone sends you a link and you go straight to the oracle, we need a default back button. If this is not the case (which it should be since it's an electron app) then we should never set this in the oracle. We set it on *the page we were Prior** to navigating to the oracle/topic.

Reply:
@alex-cory that makes sense 👍 I was wondering if we were going to take the breadcrumb approach and just keep a history of everywhere the user has been. but that doesn't make real sense since we dont have a global back button.

Reply:
we can eventually do something like that, but for now I think this is actually a decent approach